app/views/supplies/new.html.haml

```diff
@@ -0,0 +1,5 @@
+= bootstrap_form_for @supply do |f|
+  %fieldset
+    = f.text_field :name
+    = f.text_field :shortcode
+  %button.btn.btn-primary Save
```

Comment:
I'm curious if there is anything specific to `bootstrap_form_for` that would have you use `%button` here instead of
```haml
= f.submit 'Save', class: 'btn btn-primary'
```
I know this renders an `input type="submit"` instead of `button`, but since there is no real inner/child content outside of text, they're effectively the same.

Or are they just doing the same thing because a `%button` in a form will submit the form, and it doesn't matter?